socialhome/streams/app/components/AuthorBar.vue

```diff
@@ -46,41 +40,66 @@
 <script>
     import Vue from "vue"
 
+
     export default Vue.component("author-bar", {
-        props: {
-            handle: {type: String, required: true},
-            name: {type: String, required: true},
-            guid: {type: String, required: true},
+        props:    {
```

Comment:
Hmm please no extra spaces to align things - it makes diffs annoying when the longest element changes and it looks weird.

Need to add eslint and flake to travis so that builds fail on code style issues - better for contributors also to get instant feedback 👍